client/android/src/org/amnezia/vpn/Prefs.kt

```diff
@@ -29,7 +29,7 @@ object Prefs {
             return sharedPreferences
         } catch (e: Exception) {
             Log.e("Android-Prefs", "Getting Encryption Storage failed, plaintext fallback")
-            return context.getSharedPreferences("com.amnezia.vpn.prefrences", Context.MODE_PRIVATE)
+            return context.getSharedPreferences("com.amnezia.vpn.preferences", Context.MODE_PRIVATE)
```

Comment:
This change is probably a bad idea since it's changing a compatibility shim. I'll drop...